resources/js/routes.js

```diff
@@ -68,12 +68,18 @@ const router = new Router({
     ],
 });
 
+const urlParams = new URLSearchParams(window.location.search);
+
+if (urlParams.has('authenticated')) {
```

Comment:
This is simply to tell the system we should be authenticated and it'll check.
If this is used illegitimately it will have no impact and you will be sent to the login page.

Reply:
FYI
I removed this before the merge because the vue3 front-end no longer uses `Vue.$storage.set('authenticated', true)` to track authentication. I'm going to replace it with something similar that matches the new version of the front-end.